gui/webdisplay/src/RWebWindowsManager.cxx

```diff
@@ -154,6 +155,43 @@ void RWebWindowsManager::AssignWindowThreadId(RWebWindow &win)
    }
 }
 
+//////////////////////////////////////////////////////////////////////////////////////////
+/// If ROOT_LISTENER_SOCKET variable is configured,
+/// message will be sent to that unix socket
+
+bool RWebWindowsManager::InformListener(const std::string &msg)
+{
+#ifdef R__WIN32
```

Comment:
Why not Win32? `TSocket` works on Windows...

Reply:
listener used only together with `rootssh`. 
In this case small messages are written to unix socket (not TCP) when web window is shown.
I have no idea if unix sockets are working on Windows?
And if one can use `ssh windows_node`.

Reply:
OK, thanks for the info. That would be interesting to try with ssh on Windows, but it's not high priority